src/components/Sum.jsx

```diff
@@ -1 +1,20 @@
 // Don't forget to add default values an check types of props
+import React from 'react';
+
+const Sum = ({ a, b }) => (
+  <p>
+    {'Sum of '}
+    {a}
+    {' and '}
+    {b}
+    {' is '}
+    {a + b}
+  </p>
+);
+
+Sum.defaultProps = {
```

Comment:
Apart from default values, you also need to check the data type of props.

Reply:
ok)